content/post/practical_ml.md

```diff
@@ -0,0 +1,77 @@
+---
+author: Ilan Man
+title: "Practical tips for real-world Data Science"
+publishDate: "2018-05-15"
+draft: true
+tags: 
+ - datascience
+---
+
+Many Data Scientists today come from a hard science background - statistics, math, physics. Hard sciences have a bias towards empirical and objective truths: a correct answer exists and we can find it by employing the scientific method, usually manifested by a formulaic approach. While not a controversial statement in itself, many years of studying and application of such a mantra can collide with the realities of the Business world. There, it becomes increasingly difficult to perfectly apply the theory. As a result, the practitioner should understand how to adjust their model and their approach accordingly.
+
+In this post we'll discuss a few considerations that the Data Scientist should note during the model building and deployment process. As a running example to help illustrate this:
+
+<ul>
+<li>Imagine we work for an events platform in which a user can send an event (e.g. "happy birthday" or "please join us for cocktail hour") to their guests using a web application</li>
+<li>We are tasked with building a classifier that, given a set of trained examples, can accurately classify a given event</li>
+<li>We cannot ask the user questions about the purposes of their event :)</li>
+</ul>
+
+The key question answered by this model is "What kind of events are our users sending?" If you work in the events space, you can imagine that answering this question is critical for the Business. Marketing can better target ads, Strategy can understand where there are holes in the marketplace (e.g. What if 95% of users sent Christmas cards only?) and Business Development can identify partnership opportunities.
```

Comment:
You could consider scrapping this paragraph and switching in a paragraph summarizing your post. E.g.,

> Practical Data Science means orienting your work to activities that drive the most business value. In this post I'll talk about three themes I use to deliver high-value data products to the business:
> * Understand the business need
> * Listen to your surroundings
> * Beware the new hotness

Reply:
Agreed - I like this short bulleted style summary.